### ISSUE TITLE
How to deal with unequally spaced data points?

### ISSUE THREAD

**Issue by qzyu999** [GAA-UAM/scikit-fda#591]
**Is your feature request related to a problem? Please describe.**
I understand that we can use this library when data points are aligned on the x-axis. What if the data points aren't equally spaced (e.g., one point at each x-interval)? Furthermore, what if there's an unequal number of points?

**Describe the solution you'd like**
I'm not sure what to do in such a case.

**Describe alternatives you've considered**
I read about PACE (https://www.stat.ucdavis.edu/PACE/), but I'm not sure whether this is the solution.

**Additional context**
Looking for some direction, my knowledge of FDA is limited.

**Comment by vnmabus** [comment]
We are working in #536 on a structure to deal with this kind of data. The basic implementation is almost finished and will be merged shortly to the develop branch. However, for that kind of data many basic operations, such as taking the mean, are not well defined. Thus, we want to provide also tools to convert this kind of data to other representations (basis and dense grid), that are better supported.

**Comment by qzyu999** [comment]
Looks like it has been merged in #536, but I am not personally testing this currently. Will close issue for clean up.

**Comment by vnmabus** [comment]
You are right, it was done in #536. I will close it as completed instead of not planned. Thanks!
